dev/README_RELEASE_AIRFLOW.md

```diff
@@ -40,7 +39,7 @@
   - [Publish release to SVN](#publish-release-to-svn)
   - [Prepare PyPI "release" packages](#prepare-pypi-release-packages)
   - [Update CHANGELOG.md](#update-changelogmd)
-  - [Manually prepare production Docker Image](#manually-prepare-production-docker-image-1)
+  - [Manually prepare production Docker Image](#manually-prepare-production-docker-image)
```

Comment:
@kaxil @jhtimmins  The main reason it was twice, because those were two different images (first was RC and the second was final one) :). I see tha tthe RC was prepared and pushed eventually anyway, But at least we should link to the common step rather than remove it from "RC" part as it is good for users to verify (I do it as well)

Reply:
Nope, then make it 3 :)

We already have another section for oushing after Vote too

Reply:
Check under RC section and then check under Publish final version :) - so definitely not a good reason ;)

Reply:
https://github.com/astronomer/airflow/blob/067738926c75148b21a41d94b48fcb5c57b9d0db/dev/README_RELEASE_AIRFLOW.md#prepare-production-docker-image

https://github.com/astronomer/airflow/blob/067738926c75148b21a41d94b48fcb5c57b9d0db/dev/README_RELEASE_AIRFLOW.md#manually-prepare-production-docker-image

Reply:
:D

Reply:
RIGHT